handlers/bot_handler.py

```diff
@@ -23,16 +24,10 @@ class IntroCommand(Command):
     @classmethod
     def execute(cls, slack_wrapper, args, timestamp, channel_id, user_id, user_is_admin):
         """Execute the Intro command."""
-        try:
-            with open("intro_msg") as f:
-                message = f.read()
-
-            slack_wrapper.post_message(channel_id, message)
-        except:
-            message = "Sorry, I forgot what I wanted to say (or the admins forgot to give me an intro message :wink:)"
-
-            slack_wrapper.post_message(channel_id, message)
+        with open("./config.json") as f:
+            message = json.load(f).get("intro_message")
```

Comment:
Have you tested this without an intro_message set? if it isn't set at all then `message` will be `None`.

We can either force us to set it (don't use `.get`) or allow it to be unset, assume empty, in which case just confirm that you tested it 'cause I am not sue if `post_message` works with `None`